### PR TITLE
NodeGraph: Add docs

### PR DESCRIPTION
**Pull request by aocenas**
Draft of the docs for the new panel https://github.com/grafana/grafana/pull/29706
Images: https://github.com/grafana/website/pull/3265

### CODE REVIEW

File: docs/sources/panels/visualizations/node-graph.md
Comment (aocenas) [inline docs/sources/panels/visualizations/node-graph.md:37]: @marcusolsson there was a discussion whether it makes sense to have this technical section here. No other panel has that as far as i see but maybe this is only panel that needs very specific naming and structure. also cc @torkelo whether this makes sense (even though I know we may change it in the next release to something more simple)

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Please move the image into a folder with a descriptive name (such as panels or node-graph) and follow the guidelines for images found in the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#images), particularly the file naming conventions.

I am currently working on a commit for this PR, should be done before too long.

**Comment by oddlittlebird** [comment]
My commits are done. Please note that I added a few comments directly to you in the text.

File: docs/sources/panels/visualizations/node-graph.md
Comment (oddlittlebird) [reply docs/sources/panels/visualizations/node-graph.md]: The panel is in beta right now. I suggest we keep everything together for now and possibly move the API content somewhere more logical when the panel finishes its beta period.

File: docs/sources/panels/visualizations/node-graph.md
Comment (aocenas) [inline docs/sources/panels/visualizations/node-graph.md:24]: @oddlittlebird I am bit unsure about what sort of definition should be here. Feel like Edge and Node are basic english words and in this context they can represent different things, it will depend on the data source and what data it returns. So you cannot say that Node mean Application or Service because that won't be always true.

**Comment by aocenas** [comment]
Added images PR https://github.com/grafana/website/pull/3265

File: docs/sources/panels/visualizations/node-graph.md
Comment (oddlittlebird) [reply docs/sources/panels/visualizations/node-graph.md:22]: Maybe something like:

A _node_ is displayed as a circle. A node might represent an application, a service, or anything else in a data network that you want to monitor.

An _edge_ is displayed as a line that connects two nodes. The connection might be a THING or OTHER THING, some relationship between the two nodes.

File: docs/sources/panels/visualizations/node-graph.md
Comment (oddlittlebird) [inline docs/sources/panels/visualizations/node-graph.md:13]:
```suggestion
![Node graph panel](img/docs/node-graph/node-graph-7-4.png "Node graph")
```

File: docs/sources/panels/visualizations/node-graph.md
Comment (oddlittlebird) [inline docs/sources/panels/visualizations/node-graph.md:34]:
```suggestion
![Node graph navigation](/img/docs/node-graph/node-graph-navigation.gif "Node graph navigation)
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

**Comment by oddlittlebird** [comment]
Image links are updated and tested. All that's left is the one outstanding conversation.

File: docs/sources/panels/visualizations/node-graph.md
Comment (oddlittlebird) [inline docs/sources/panels/visualizations/node-graph.md:21]:
```suggestion
- A _node_ is displayed as a circle. A node might represent an application, a service, or anything else that is relevant from an application perspective.
```

File: docs/sources/panels/visualizations/node-graph.md
Comment (oddlittlebird) [inline docs/sources/panels/visualizations/node-graph.md:22]:
```suggestion
- An _edge_ is displayed as a line that connects two nodes. The connection might be a request, an execution, or some other relationship between the two nodes.
```

**Review by oddlittlebird — APPROVED** [review]
LGTM once those last two suggestions are merged.

**Pull request merged**
